api/turing/validation/validator.go

```diff
@@ -175,24 +214,55 @@ func validateRouterConfig(sl validator.StructLevel) {
 	}
 
 	// Validate traffic rules
+	allRuleRoutesSet := set.New()
 	if routerConfig.TrafficRules != nil {
 		if len(routerConfig.TrafficRules) > 0 {
-			checkDefaultTrafficRule(sl, "DefaultTrafficRule", routerConfig.DefaultTrafficRule)
```

Comment:
While this check is logically correct, do you think it'll be better to move it into a separate check: `validateDefaultRouteTrafficRules(routeID, rules, defaultRoute)`

Within that function, you can append the default rule to the other rules and run the check to make sure that routeID appears in each rule. That will be clearer and better isolated, right?

```golang
if routerConfig.TrafficRules != nil {
...
    if routerConfig.DefaultTrafficRule != nil {
        validateDefaultRouteTrafficRules(routeID, rules, defaultRoute)
```

wdyt?

Reply:
Would you mean to shift the newly added logic to `checkDefaultTrafficRule` into a new function called `validateDefaultRouteTrafficRules `?

Reply:
Yes!